addons/product/models/product_template.py

```diff
@@ -1108,3 +1108,20 @@ def get_import_templates(self):
             'label': _('Import Template for Products'),
             'template': '/product/static/xls/product_template.xls'
         }]
+
+    @api.multi
+    def _compute_message_attachment_count(self):
+        super(ProductTemplate, self)._compute_message_attachment_count()
```

Comment:
```suggestion
        super()._compute_message_attachment_count()
```
(Reference from https://github.com/odoo/odoo/pull/33975#discussion_r291676373)